.circleci/config.yml

```diff
@@ -21,7 +21,7 @@ jobs:
           name: Install dependencies
           command: |
             apt install -yy python3-venv
-            pip3 install poetry
+            pip3 install poetry==0.12.11 # https://github.com/sdispater/poetry/issues/1060 can probably be unpinned when you read this.
```

Comment:
I read this, but I don't think it can be unpinned :stuck_out_tongue:

Reply:
Left it for the next person to change `.circleci` :)